From 8f42f4ffe61723135abb6b9c9d1d9b1dc9013de1 Mon Sep 17 00:00:00 2001
From: Geert Uytterhoeven <geert+renesas@glider.be>
Date: Fri, 23 May 2014 09:46:20 +0200
Subject: ARM: shmobile: r8a7779 dtsi: Update unit-addresses for clocks

  - Correct the unit-address for the "cpg_clocks" node,
  - Add missing unit-addresses for the "mstp*_clks" nodes,
  - Rename "cpg_clocks" and "mstp*_clks" nodes to the more generic
    "clocks".

Signed-off-by: Geert Uytterhoeven <geert@linux-m68k.org>
Acked-by: Laurent Pinchart <laurent.pinchart@ideasonboard.com>
Signed-off-by: Simon Horman <horms+renesas@verge.net.au>
(cherry picked from commit 2909b8746d6f5534aa7714d2df110b3633a59556)
Signed-off-by: Simon Horman <horms+renesas@verge.net.au>
---
 arch/arm/boot/dts/r8a7779.dtsi | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/arch/arm/boot/dts/r8a7779.dtsi b/arch/arm/boot/dts/r8a7779.dtsi
index 038c16a18373..27bf2274ff08 100644
--- a/arch/arm/boot/dts/r8a7779.dtsi
+++ b/arch/arm/boot/dts/r8a7779.dtsi
@@ -298,7 +298,7 @@
 		};
 
 		/* Special CPG clocks */
-		cpg_clocks: cpg_clocks@0xe6150000 {
+		cpg_clocks: clocks@ffc80000 {
 			compatible = "renesas,r8a7779-cpg-clocks";
 			reg = <0 0xffc80000 0 0x30>;
 			clocks = <&extal_clk>;
@@ -342,7 +342,7 @@
 		};
 
 		/* Gate clocks */
-		mstp0_clks: mstp0_clks {
+		mstp0_clks: clocks@ffc80030 {
 			compatible = "renesas,r8a7779-mstp-clocks",
 			             "renesas,cpg-mstp-clocks";
 			reg = <0 0xffc80030 0 4>;
@@ -379,7 +379,7 @@
 				"scif1", "scif0", "i2c3", "i2c2", "i2c1",
 				"i2c0";
 		};
-		mstp1_clks: mstp1_clks {
+		mstp1_clks: clocks@ffc80034 {
 			compatible = "renesas,r8a7779-mstp-clocks",
 			             "renesas,cpg-mstp-clocks";
 			reg = <0 0xffc80034 0 4>, <0 0xffc80044 0 4>;
@@ -408,7 +408,7 @@
 				"ether", "sata",
 				"pcie", "vin3";
 		};
-		mstp3_clks: mstp3_clks {
+		mstp3_clks: clocks@ffc8003c {
 			compatible = "renesas,r8a7779-mstp-clocks",
 			             "renesas,cpg-mstp-clocks";
 			reg = <0 0xffc8003c 0 4>;
-- 
2.1.2

